.yarnrc.yml

```diff
@@ -15,4 +15,4 @@ supportedArchitectures:
     - darwin
     - win32
 
-yarnPath: .yarn/releases/yarn-3.8.6.cjs
+yarnPath: .yarn/releases/yarn-4.7.0.cjs
```

Comment:
I was able to get `corepack` to work locally by running `corepack enable` and removing this line. That lets us remove the `.yarn/releases/yarn-4.7.0.cjs` file. Did it not work for you?

Reply:
You're right it's working for me now

Reply:
I'm going to continue to look into this after lunch, but it looks like if you don't mention which yarn to use, it defaults to the global version, which could cause errors if we have projects on different yarn versions. Since non-orgs will be using it, it's prob best to keep the version of yarn in the repo.

Reply:
If I run `corepack disable` then `yarn` it prints this error:

```
error This project's package.json defines "packageManager": "yarn@4.7.0". However the current global version of Yarn is 1.22.22.

Presence of the "packageManager" field indicates that the project is meant to be used with Corepack, a tool included by default with all official Node.js distributions starting from 16.9 and 14.19.
Corepack must currently be enabled by running corepack enable in your terminal. For more information, check out https://yarnpkg.com/corepack.
```

So I don't think it will accidentally let you use yarn v1 if you have `packageManager` set to v4.